rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -207,7 +207,7 @@ def compute_and_clip_gradients(policy: Policy, optimizer: LocalOptimizer,
     # Clip by global norm, if necessary.
     if policy.config["grad_clip"] is not None:
         # Defuse inf gradients (due to super large losses).
-        grads = [g for (g, v) in grads_and_vars]
+        grads = [g for (g, _) in grads_and_vars if g is not None]
```

Comment:
why do we need this?
we have the following right below.
```
policy.grads = [
            tf.where(tf.math.is_nan(g), tf.zeros_like(g), g) for g in grads
        ]
```
also this will change the shape of the gradient vector, isn't it?

Reply:
The gradient is None NaN, which means that the gradient was not event initialized so the shape of the gradient vector will not be changed. Right?